build_all/linux/build.sh

```diff
@@ -7,7 +7,6 @@ set -e
 script_dir=$(cd "$(dirname "$0")" && pwd)
 build_root=$(cd "${script_dir}/../.." && pwd)
 run_unit_tests=ON
-use_wsio=OFF
 run_valgrind=0
 build_folder=$build_root"/cmake/uamqp_linux"
 run_unittests=OFF
```

Comment:
Gotta remove this too:

build_all\linux\build.sh:21:    echo " --use-websockets              Enables the support for AMQP over WebSockets."